common/templates/version.cpp.in

```diff
@@ -0,0 +1,14 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include "claragenomics/version.hpp"
+
+std::string VERSION = "@VERSION@";
+std::string VERSION_SHORT = "@VERSION_SHORT@";
```

Comment:
Even though this is a file that needs to be configured I would place it with the rest of the source files in `base/src`.